makezip.sh

```diff
@@ -6,8 +6,9 @@ TMP_DIR=`mktemp -d`
 
 pushd $TMP_DIR
 pip install -r $CURRENT_DIR/requirements.txt -t .
-cp -ra $CURRENT_DIR/app/. .
+cp -ra $CURRENT_DIR/app/* .
 cp -rp $CURRENT_DIR/app/.env.production .env
+find . -name \*.pyc -delete
```

Comment:
could also avoid generating the pyc files by setting a `PYTHONDONTWRITEBYTECODE=1` env var.